demisto_sdk/commands/content_graph/content_graph_commands.py

```diff
@@ -78,9 +80,32 @@ def update_content_graph(
         content_graph_interface.clean_import_dir()
         if not imported_path:
             # getting the graph from remote, so we need to clean the import dir
-            extract_remote_import_files(content_graph_interface, builder)
-
+            try:
+                extract_remote_import_files(content_graph_interface)
+            except RuntimeError as e:
+                logger.warning(
+                    "Failed to download the content graph, will create a new graph"
+                )
+                logger.debug(f"Runtime Error: {e}")
```

Comment:
won't it show only `Failed to download the content graph` and mask the original error message?
maybe we can go with something similar to the suggestion above with the import (success = ...)

Reply:
We will see the original error message in DEBUG, we don't care about the original message in INFO